server/src/main/java/org/opensearch/search/aggregations/metrics/AbstractPercentilesAggregationBuilder.java

```diff
@@ -362,7 +362,9 @@ PercentilesConfig configOrDefault() {
 
     @Override
     protected XContentBuilder doXContentBody(XContentBuilder builder, Params params) throws IOException {
-        builder.array(valuesField.getPreferredName(), values);
+        if (valuesField != null) {
```

Comment:
Why this change?

Reply:
I had added this check because of failing test. Not able to reproduce it and hence removing this check.